src/main/java/uk/gov/hmcts/juror/api/moj/service/JurorHistoryServiceImpl.java

```diff
@@ -237,8 +237,12 @@ public void createSummonsLetterHistory(JurorPool jurorPool, String otherInfo) {
         registerHistorySystem(jurorPool, HistoryCodeMod.PRINT_SUMMONS, otherInfo);
     }
 
-    public void createJuryAttendanceHistory(JurorPool jurorPool, String otherInfo) {
-        registerHistorySystem(jurorPool, HistoryCodeMod.JURY_ATTENDANCE, otherInfo);
+    public void createJuryAttendanceHistory(JurorPool jurorPool, String otherInfo, String username) {
+        registerHistory(jurorPool, HistoryCodeMod.JURY_ATTENDANCE, otherInfo, username);
```

Comment:
We should have a method to pull the username from SecurityUtil so we don't need to pass it in here